config/localTemplate.go

```diff
@@ -452,6 +452,10 @@ type Local struct {
 	// MaxAcctLookback sets the maximum lookback range for account states,
 	// i.e. the ledger can answer account states questions for the range Latest-MaxAcctLookback...Latest
 	MaxAcctLookback uint64 `version[23]:"4"`
+
+	// EnableSyncMode will boot the node with a subset of services, and allow users to control which rounds
+	// are synced to the ledger enabling them to ensure desired deltas are kept in the cache.
+	EnableSyncMode bool `version[23]:"false"`
```

Comment:
Does this need to be 24? I'm not sure how the config file versioning works.

Reply:
Yes, I think I updated this prior to the most recent changes being committed--I'm not intending to decrease the version.